app/logic/Calendar/Event.ts

```diff
@@ -194,6 +198,29 @@ export class Event extends Observable {
     this.response = original.response;
   }
 
+  get outgoingInvitation() {
+    return new OutgoingInvitation(this);
+  }
+
+  protected participantMe(): { identity: MailIdentity, participant: Participant, person: PersonUID } {
+    let results = [];
+    for (let account of appGlobal.emailAccounts) {
+      if (!account.canSendInvitations) {
+        continue;
+      }
+      for (let identity of account.identities) {
+        for (let participant of this.participants) {
+          if (identity.isEMailAddress(participant.emailAddress)) {
+            let person = new PersonUID(participant.emailAddress, identity.realname);
+            results.push({ identity, participant, person });
```

Comment:
Why not directly `return {}` here? Given that you return the first only anyway?

Reply:
Only if it's unique.

Reply:
I thought you just wanted fail, if there are none found.
You're actually also failing, if there are several matching.
I am not sure what the caller is supposed to do with that error.